scripts/enable_lambdas.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import ast
+from azul import config
+from azul.types import JSON
+import boto3
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RedButton:
+    def __init__(self):
+        self.azul_lambda = boto3.client('lambda')
+
+    def enable_azul_lambdas(self, enabled: bool):
+        paginator = self.azul_lambda.get_paginator('list_functions')
+        lambda_prefixes = [config.qualified_resource_name(lambda_infix) for lambda_infix in config.lambda_names()]
+        assert all(lambda_prefixes)
+        for lambda_page in paginator.paginate(FunctionVersion='ALL', MaxItems=500):
+            for lambda_name in [metadata['FunctionName'] for metadata in lambda_page['Functions']]:
+                if any(lambda_name.startswith(prefix) for prefix in lambda_prefixes):
+                    lambda_settings = self.azul_lambda.get_function(FunctionName=lambda_name)
+                    lambda_arn = lambda_settings['Configuration']['FunctionArn']
+                    lambda_tags = self.azul_lambda.list_tags(Resource=lambda_arn)['Tags']
+                    if enabled:
+                        self.enable_lambda(lambda_settings, lambda_tags)
+                    else:
+                        self.disable_lambda(lambda_settings, lambda_tags)
+
+    def enable_lambda(self, lambda_settings: JSON, lambda_tags: JSON):
+        lambda_name = lambda_settings['Configuration']['FunctionName']
+        if 'azul-original-concurrency-limit' in lambda_tags.keys():
```

Comment:
Extract a class attribute

```suggestion
        if self.tag_name: in lambda_tags.keys():
```